ethereum/contracts/utils/MMRProofVerification.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.5;
+
+struct MMRProof {
+    bytes32[] items;
+    uint64 order;
```

Comment:
maybe add to comment to explain that order is a bitfield, since it's no longer in the variable name